### PR TITLE
E_CLASSROOM-295 [Admin] [FE/BE] Admin User search by name or email

### PR DESCRIPTION
**Pull request by ErickDoroy**
### Issue Link
- https://framgiaph.backlog.com/view/E_CLASSROOM-295
### Definition of Done
- [x] Can search using name or email

## Related PR:
- https://github.com/framgia/sph-classroom-els-be/pull/69
#### Main note
- http://localhost:3003/admin/users?page=1&search=
#### Pages Affected
- Admin Account

### Scenarios/ Test Cases
- [ ] Able to search by name or email
#### User Interface
- N/A
#### User Experience 
- N/A
#### Functionality
- N/A
### Screenshots
adminsearch

### CODE REVIEW

**Comment by catampongan** [comment]
## Approved PR #136 
- [x] Able to search either name or email
-----------------------
task-295

-------------------
- Total Test Time: 0.25 hr
- Total Iterations: 0
- Total Comments: 0

File: src/pages/admin/Admin/AdminList/index.js [hunk 1]
```diff
@@ -38,13 +41,19 @@ const AdminList = () => {
...
 
   const load = () => {
-    AdminApi.getAdminAccounts({ page })
+    AdminApi.getAdminAccounts({ 
+      page,
+      search
+    })
       .then(({ data }) => {
         setAdminAccounts(data.data);
+        console.log(data.data);
```
Comment (karlolee50) [inline src/pages/admin/Admin/AdminList/index.js]: remove if not used

File: src/pages/admin/Admin/AdminList/index.js [hunk 2]
```diff
@@ -82,11 +96,21 @@ const AdminList = () => {
...
                 type="text"
+                aria-label="Search"
+                value={search}
+                onChange={(e) => {
+                  setSearch(e.target.value);
+    
+                  if (e.target.value.length === 0) {
+                    setPage(1);
+                    setSearchStatus(!searchStatus);
+                  }
```
Comment (karlolee50) [inline src/pages/admin/Admin/AdminList/index.js]: put this in a function

**Review by karlolee50 — CHANGES_REQUESTED** [review]
Review time: 5 minutes
Comment/s: 2

**Review by karlolee50 — APPROVED** [review]

**Pull request merged**